### ISSUE TITLE
Timepicker is not keyboard accessible

### ISSUE THREAD

**Issue by stephenharris** [stephenharris/Event-Organiser#281]
Although a time could be entered manually, the timepicker UI is not keyboard accesible. This should be rectified, and behaviour in a similar way to jQuery's datepicker. Suggested shortcuts:

 - `esc` - close
 - `ctrl/cmd + home` - open
 - `ctrl/cmd + up/right/down/left` - navigate hour grid
 - `shift + up/right/down/left` - navigate minute grid
 - `enter` - use selected hour & minute
